src/java.base/share/classes/java/lang/ref/ReferenceQueue.java

```diff
@@ -142,26 +201,15 @@ public Reference<? extends T> poll() {
      *          If the timeout wait is interrupted
      */
     public Reference<? extends T> remove(long timeout)
-        throws IllegalArgumentException, InterruptedException
-    {
-        if (timeout < 0) {
-            throw new IllegalArgumentException("Negative timeout value");
-        }
-        synchronized (lock) {
-            Reference<? extends T> r = reallyPoll();
-            if (r != null) return r;
-            long start = (timeout == 0) ? 0 : System.nanoTime();
-            for (;;) {
-                lock.wait(timeout);
-                r = reallyPoll();
-                if (r != null) return r;
-                if (timeout != 0) {
-                    long end = System.nanoTime();
-                    timeout -= (end - start) / 1000_000;
-                    if (timeout <= 0) return null;
-                    start = end;
-                }
-            }
+        throws IllegalArgumentException, InterruptedException {
+        if (timeout < 0) throw new IllegalArgumentException("Negative timeout value");
+        if (timeout == 0) return remove();
```

Comment:
Nit: use the same formatting as in `NativeReferenceQueue::remove` and in this file.

```
        if (timeout < 0)
            throw new IllegalArgumentException("Negative timeout value");
        if (timeout == 0)
            return remove();
```

Reply:
okay